src/some-package/java/com/mainpackage/FileSyntaxError.java

```diff
@@ -0,0 +1,37 @@
+package com.mainpackage;
+
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.model.S3Object;
+import java.io.IOException;
+import java.io.InputStream;
+import com.amazonaws.util.IOUtils;
+
+/**
+ * Even though this file contains weak crypto issues, CodeGuru Reviewer will not
+ * report any issues in it, because it has been excluded in aws-codeguru-reviewer.yml.
+ * 
+ * For more information, see the Amazon CodeGuru Reviewer User Guide:
+ * https://docs.aws.amazon.com/codeguru/latest/reviewer-ug/welcome.html.
+ */
+public class FileSyntaxError {
+
+    // {fact rule=file-syntax-error@v1.0 defects=1}
+    public void getObjectContentNoncompliant(AmazonS3 amazonS3Client, String bucketName, String key) throws IOException {
+        final S3Object s3object = amazonS3Client.getObject(bucketName, key);
+        // Noncompliant: the statement is incomplete and is missing ";" at the end.
+        System.out.println(s3object.getObjectMetadata())
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Syntax error, insert &#34;;&#34; to complete Statement

This file contains syntax errors or language features not currently supported by CodeGuru Reviewer in Java versions greater than Java 8, and can&#39;t be analyzed by all of CodeGuru Reviewer&#39;s detectors.